hatch_build.py

```diff
@@ -434,6 +434,8 @@
     "tabulate>=0.7.5",
     "tenacity>=8.0.0,!=8.2.0",
     "termcolor>=1.1.0",
+    # uuid6 is necessary to use UUID7 in our models
+    "uuid6",
```

Comment:
This is now already added on line 442 -- so can be removed

https://github.com/apache/airflow/blob/51f9e835bf4e40e418f9f45a510a639068b08fa2/hatch_build.py#L441